packages/context/src/__tests__/unit/context-view.unit.ts

```diff
@@ -45,6 +45,17 @@ describe('ContextView', () => {
     expect(await taggedAsFoo.asGetter()()).to.eql(['BAR', 'FOO']);
   });
 
+  it('reports error on singleValue() if multiple values exist', async () => {
+    return expect(taggedAsFoo.singleValue()).to.be.rejectedWith(
```

Comment:
I'm curious why we wanted to have the `singleValue()` method in the first place?

Reply:
It's for dynamic configuration - `@config.view`. See acceptance tests for refs.

Reply:
Okay, but we can still use `values()` for that too no?

Reply:
Yes, we can still use `values()`. `singleValue()` is just a helper.